src/mds/Server.cc

```diff
@@ -1148,6 +1150,9 @@ void Server::find_idle_sessions()
 		    "has arrived" << dendl;
 	// evict session without marking it stale
 	to_evict.push_back(session);
+	if (any_osd_laggy) {
+	  laggy_sessions.insert(session);
```

Comment:
```suggestion
	  laggy_sessions.insert(session->get_client());
```

etc.

Reply:
okay, will change set type to `client_t`, had done it as per https://github.com/ceph/ceph/pull/49971#discussion_r1131490343

Reply:
done